llama_hub/file/pdf_plumber/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Init file."""
+from llama_hub.file.pdf_miner.base import (
```

Comment:
should this be `pdf_plumber` instead of `pdf_miner`?

Reply:
Whoops, yep! Copied and changed than init file. Missed that. Thanks! Going to fix now.

Reply:
Thanks!!! Merging now :)